include/database_functions.php

```diff
@@ -205,15 +151,58 @@ function insertClass($type, $link, $classDate, $studentId){
 ");
 }
 
+//If there is a better way to handle this please let me know. So, I have a credits table (Every row in that table in actually a purchase the student made).If
+//we subtract a credit means we are returning money to the person which is a refund. So, I have a refund table whith the date of the refund and the amount. 
+//The value will only be inserted in the table if the student has a remaing credit. So, before this query happens I need to check if the student has remaining credits
+//But also, to calculate the remaining credits I need to check if there is any refund in the table for that student. 
+function insertRefund($amount, $studentId){    
+    dbQuery("
+    INSERT INTO refunds(Amount, StudentId)
+    VALUES ('$amount', '$studentId')
+");
+}
+
 // Deletions
 // -------------------------------------------------------------------------- 
 
-function deleteStudents_TableDB($studentId){    
+function deleteStudent($studentId){    
     dbQuery("
         DELETE FROM students WHERE StudentId = $studentId
 ");
 }
 
+function deleteClass($classId){    
+    dbQuery("
+        DELETE FROM classes WHERE ClassId = $classId
```

Comment:
Sounds great! I will do this for students and classes. What I'm thinking is that since credits do not get deleted, but refunded, then I won't put it on the credits table.

Reply:
When doing this I added a where clause (WHERE DateArchived is NULL) to retrieve data. I did not do it in all queries here because I got rid of most of them in the Jog 4. So I only added it to the ones I currently have in the current state on my code.

Reply:
Sounds good to me!